WebContent/register.jsp

```diff
@@ -21,7 +21,91 @@
 	<div class="container">
 		<div class="row"
 			style="margin-top: 5px; margin-left: 2px; margin-right: 2px;">
+			
+			<label for="userType">Select User Type:</label>
+    			<select id="userType" class="col-md">
+        			<option value="company">Company</option>
+        			<option value="student" selected>Student</option>
+    			</select>
+    			
+    		  <!-- Fields for Company Registration -->
+    		<div id="companyFields" style="display: none;">
+        		<form action="./RegisterSrv" method="post"
+				class="col-md-6 col-md-offset-3"
+				style="border: 1px grey; border-radius: 10px; background-color: #ffffff; padding: 25px;">
+				<div style="font-weight: bold;" class="text-center">
+					<h2 style="color: black;">Registration Form</h2>
+					<%
+					if (message != null) {
+					%>
+					<p style="color: blue;">
+						<%=message%>
+					</p>
+					<%
+					}
+					%>
+				</div>
+				<div></div>
+				<div class="row">
+					<input type="hidden" id="userTypeHidden" name="userType" value="company">
+					<div class="col-md-6 form-group">
+						<label for="first_name">Company Name</label> <input type="text"
+							name="companyName" class="form-control" id="first_name"
+							name="first_name" required>
+					</div>
+					<div class="col-md-6 form-group">
+						<label for="first_name">Representative's Name</label> <input type="text"
+							name="username" class="form-control" id="first_name"
+							name="first_name" required>
+					</div>
+					<div class="col-md-6 form-group">
+						<label for="first_name">Email</label> <input type="email"
+							name="email" class="form-control" id="first_name"
+							name="first_name" required>
+					</div>
+				</div>
+				<div class="form-group">
+					<label for="last_name">Address</label>
+					<textarea name="address" class="form-control" id="last_name"
+						name="last_name" required></textarea>
+				</div>
+				<div class="row">
+					<div class="col-md-6 form-group">
+						<label for="last_name">Phone number</label> <input type="number"
+							name="mobile" class="form-control" id="last_name"
+							name="last_name" required>
+					</div>
+					<div class="col-md-6 form-group">
+						<label for="last_name">Pin Code</label> <input type="number"
+							name="pincode" class="form-control" id="last_name"
+							name="last_name" required>
+					</div>
 
+				</div>
+				<div class="row">
+					<div class="col-md-6 form-group">
+						<label for="last_name">Password</label> <input type="password"
+							name="password" class="form-control" id="last_name"
+							name="last_name" required>
```

Comment:
can you remove these ``name="last_name" please, it's not really useful

Reply:
lets do that later as its on every .jsp page